torch/_logging/_registrations.py

```diff
@@ -3,6 +3,7 @@
 register_log("dynamo", "torch._dynamo")
 register_log("aot", "torch._functorch.aot_autograd")
 register_log("inductor", "torch._inductor")
+register_log("perf_hints", "torch._inductor.perf_hints")
```

Comment:
This probably not work since when you define perf_hints logger in each individual file, the logger name will be something like 'torch._inductor.graph.perf_hints' and it does not match the logger name registered here. Maybe register a artifact log instead.

Even if we have a logger name mismatch here, you probably still see log messages because those messages are logged in warning level which is on by default.

Reply:
I see. Will fix.